### PR TITLE
Revise “Reconfigure a Node's Kubelet in a Live Cluster”

### PR DESCRIPTION
**Pull request by sftim**
Revise [Reconfigure a Node's Kubelet in a Live Cluster](https://k8s.io/docs/tasks/administer-cluster/reconfigure-kubelet/):
- Revise wording for version requirements
- Add glossary tooltips in summary
- Use sentence case for headings
- Write kubelet in lowercase where appropriate 
- Add “What's next” section

[[preview](https://deploy-preview-18629--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/reconfigure-kubelet/)]

/kind cleanup

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 1]
```diff
@@ -43,9 +43,9 @@ flag must be set to a writable directory.
 
 {{% capture steps %}}
 
-## Reconfiguring the Kubelet on a Live Node in your Cluster
+## Reconfiguring the kubelet on a running Node in your cluster
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md]: Does node need to be capitalized?

File: content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 reviewers:
 - mtaufen
 - dawnchen
-title: Reconfigure a Node's Kubelet in a Live Cluster
+title: Reconfigure a Node's kubelet in a Live Cluster
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md]: I'm not sure about this change. Line 5 is the title.

File: content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ if your cluster is running v1.16 then you can use kubectl v1.15, v1.16
 or v1.17; other combinations
 [aren't supported](/docs/setup/release/version-skew-policy/#kubectl).
 
-On each of the Node(s) you're reconfiguring, the Kubelet's `--dynamic-config-dir`
+On each of the Node(s) you're reconfiguring, the kubelet's `--dynamic-config-dir`
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md]: Line 40, lowercase node?
For each node that you're reconfiguring, you must set the kubelet `--dynamic-config-dir` flag to a writable directory.

File: content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 4]
```diff
@@ -101,14 +101,14 @@ and debug issues. The compromise, however, is that you must start with knowledge
...
 intend to change.
 
-Ideally, the Kubelet would be bootstrapped from a file on disk
+Ideally, the kubelet would be bootstrapped from a file on disk
 and you could edit this file (which could also be version-controlled),
-to create the first Kubelet ConfigMap
-(see [Set Kubelet parameters via a config file](/docs/tasks/administer-cluster/kubelet-config-file)),
-Currently, the Kubelet is bootstrapped with **a combination of this file and command-line flags**
+to create the first kubelet ConfigMap
+(see [Set kubelet parameters via a config file](/docs/tasks/administer-cluster/kubelet-config-file)),
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md]: @sftim , would you double check that the sentence starting on line 104 ends somewhere? I am having trouble parsing the content. Thanks.

File: content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 5]
```diff
@@ -173,7 +173,7 @@ data:
 ```
 
 The ConfigMap is created in the `kube-system` namespace because this
-ConfigMap configures a Kubelet, which is Kubernetes system component.
+ConfigMap configures a kubelet, which is Kubernetes system component.
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md]: nit: Lines 175 - 176 are hard to parse. Possible edit:
The ConfigMap, from the previous example, is created in the `kube-system` namespace.
The `kube-system` namespace is used because the kubelet is a Kubernetes system component.

File: content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md (hunk 4 above)
Comment (sftim) [reply content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md]: Reworded.

File: content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md (hunk 5 above)
Comment (sftim) [reply content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md]: Reworded.

**Comment by sftim** [comment]
BTW this PR is about style changes. I haven't checked that the content is technically accurate.

**Comment by sftim** [comment]
@jimangel rebased & squashed
